py-scripts/lf_interop_ping_plotter.py

```diff
@@ -723,7 +799,7 @@ def get_pass_fail_list(self):
                     pass_fail_list.append("FAIL")
         return pass_fail_list,test_input_list
                     
-    def generate_report(self, result_json=None, result_dir='Ping_Plotter_Test_Report', report_path=''):
+    def generate_report(self, result_json=None, result_dir='Ping_Plotter_Test_Report', report_path='', config_devices='', group_device_map={}):
```

Comment:
Mutable defaults `group_device_map={}` should be changed to `group_device_map=None` then assigned to empty dict if still None once function starts.
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments